app/controllers/application_submissions_controller.rb

```diff
@@ -55,17 +55,9 @@ def past_applications
   end
 
   def review
-    if params.require(:accepted) == 'true'
-      interview_parameters = params.require(:interview)
-                                   .permit(:location, :scheduled)
-      interview_parameters.require :location
```

Comment:
So they weren't required here because they were included in .permit?

Reply:
I'm pretty sure that someone wrote `.require` because they wanted to make sure that the parameters were definitely included.
However, if they're not included, `create!` will fail. So we already have protection against creating an interview with incorrect attributes.